indico/modules/logs/controllers.py

```diff
@@ -159,3 +161,34 @@ def object(self):
     @property
     def object_tzinfo(self):
         return get_timezone(config.DEFAULT_TIMEZONE)
+
+
+class RHResendEmail(RHManageEventBase):
+    """Resend an email log entry."""
+
+    def _process_args(self):
+        self.entry = EventLogEntry.get(request.view_args['log_entry_id'])
+        if self.entry is None:
+            raise NotFound('Event log entry does not exist.')
+        return super()._process_args()
```

Comment:
we use explicit base class names in these methods

```suggestion
        return RHManageEventBase._process_args(self)
```